src/tfs/tlog.c

```diff
@@ -29,6 +30,13 @@ static const char *tfs_magic = "NVMTFS";
 #define TFS_EXTENSION_LINK_BYTES (1 + 2 * MAX_VARINT_SIZE)
 #define TFS_LOG_RESERVED_BYTES (TFS_EXTENSION_HEADER_BYTES + TFS_EXTENSION_LINK_BYTES)
 
+/* Minimum number of obsolete log entries needed to trigger a log compaction. */
+#define TFS_LOG_COMPACT_OBSOLETE   8192
```

Comment:
move these tunables to uniboot.h?

Reply:
OK, I moved the TFS_LOG_COMPACT_OBSOLETE and TFS_LOG_COMPACT_RATIO tunables to src/config.h.